recipes/minian/meta.yaml

```diff
@@ -0,0 +1,115 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "MiniAn" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MetaCell/minian/archive/feature/83.zip
```

Comment:
This file is quite large (604 MB) and it looks like it's mostly the demo movies. It would probably be good at some point to make a release on pypi that only has the files needed to run the package assuming the demo movies aren't needed.

Reply:
removed the demo movies from the tarball and created an entrypoint command for installing the needed demo movies 
```minian-install-demo```